COMMERCE-9603 - As it's not possible to delete/remove a payment method from a channel, Delete permission checkboxes should be disabled

victorcdp opened the pull request:
Payment methods had a permission of "delete", which was useless because payment methods can't be deleted.
Proposed fix is to remove the "delete" column of the channel payment method permissions portlet table

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan123 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/2612/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2612/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 87 out of 93 jobs passed in 2 hours 31 minutes 57 seconds 105 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4dd4146f40720236b2221435ed5083ebc4b2f41a">4dd4146f40720236b2221435ed5083ebc4b2f41a</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/4d969d562de75bb681fd8576f9564b5d113da8b0">4d969d562de75bb681fd8576f9564b5d113da8b0</a></p>87 out of 93 jobs PASSED<div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest(master)/2612/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288385/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/205431/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/34</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164842/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/36</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314160/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288389/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li></ul></div><details><summary><strong>87 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51935/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291822/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314166/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291818/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312825/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164840/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291819/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314163/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/12</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254150/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/13</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252128/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/14</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164837/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/15</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291816/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/16</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254151/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/17</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51933/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/18</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252129/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/19</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51932/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312821/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/20</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254147/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/21</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291813/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/22</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254146/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/23</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254144/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/24</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252121/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/25</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254143/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/26</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252120/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/27</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312819/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/28</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252123/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/29</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51937/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/30</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252133/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/31</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303059/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/32</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288390/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/33</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312828/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/35</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254154/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/37</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/205430/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/38</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254155/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/39</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312823/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252131/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/40</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164841/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/41</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252132/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/42</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314165/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/43</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291821/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/44</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303057/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/45</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312827/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/46</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254153/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/47</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314164/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/48</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288388/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/49</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288384/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288387/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/50</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164839/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/51</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314162/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/52</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312826/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/53</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254152/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/54</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51934/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/55</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252130/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/56</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288386/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/57</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314161/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/58</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164838/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/59</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254149/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288381/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/60</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291812/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/61</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288380/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/62</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254145/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/63</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51929/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/64</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291814/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/65</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252125/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/66</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312820/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/67</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252122/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/68</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288379/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/69</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164843/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/70</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252127/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51931/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291815/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252126/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288383/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/254148/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164836/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312822/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51930/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252124/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312829/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql57-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288382/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291820/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314167/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303058/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/205429/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/51936/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291817/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312824/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2612/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288385/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></h5><div><h6>Job Results:</h6><p>13 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/288385//consoleText">AXIS_VARIABLE=0,label_exp=!master #288385</a><ol><li><div><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/288385//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Microblogs_EditEntry_">LocalFile.Microblogs#EditEntry</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.Microblogs_EditEntry/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.Microblogs_EditEntry/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/3/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/205431/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/34</a></h5><div><h6>Job Results:</h6><p>24 Tests Passed.<br/>3 Tests Failed.</p></div><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/205431//consoleText">AXIS_VARIABLE=0,label_exp=!master #205431</a><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/205431//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault_">LocalFile.DisplayPagesWithStagingAndWebContent#PublishDisplayPageMarkedAsDefault</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/205431//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage_">LocalFile.DisplayPagesWithStagingAndWebContent#PublishEditedDisplayPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/205431//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher_">LocalFile.DisplayPagesWithWebContent#ViewDisplayPageViaAssetPublisher</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/LocalFile.DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/34/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/164842/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/36</a></h5><div><h6>Job Results:</h6><p>21 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/164842//consoleText">AXIS_VARIABLE=2,label_exp=!master #164842</a><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/164842//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowDDL_EditDDLRecordViaMyWorkflowTasks_">LocalFile.WorkflowDDL#EditDDLRecordViaMyWorkflowTasks</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/LocalFile.WorkflowDDL_EditDDLRecordViaMyWorkflowTasks/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/LocalFile.WorkflowDDL_EditDDLRecordViaMyWorkflowTasks/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/164842//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowUser_CreateNewUserAccount_">LocalFile.WorkflowUser#CreateNewUserAccount</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/LocalFile.WorkflowUser_CreateNewUserAccount/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/LocalFile.WorkflowUser_CreateNewUserAccount/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/36/2/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/314160/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/314160//consoleText">AXIS_VARIABLE=0,label_exp=!master #314160</a><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/314160//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_CalendarStaging_SchedulePublishToLiveWithEventAndResource_">LocalFile.CalendarStaging#SchedulePublishToLiveWithEventAndResource</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/7/0/LocalFile.CalendarStaging_SchedulePublishToLiveWithEventAndResource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/7/0/LocalFile.CalendarStaging_SchedulePublishToLiveWithEventAndResource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1557873038878/test-portal-acceptance-pullrequest(master)/2612/functional-tomcat90-mysql57-jdk8/7/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288389/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/288389//consoleText">AXIS_VARIABLE=0,label_exp=!master #288389</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':private:apps:portal-workflow:portal-workflow-metrics-web:npmRunTest'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/build/node/lib/node_modules/npm/bin/npm-cli.js, --production, false, --registry, http://mirrors.lax.liferay.com:4873, --loglevel, warn, --progress, true, run, test]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':private:apps:portal-workflow:portal-workflow-metrics-web:npmRunTest'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)</code></pre></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
Reviewing now :)

brianikim commented:
See: https://github.com/SamZiemer/liferay-portal/pull/602

brianikim committed:
COMMERCE-8451 Use cpSpecificationOption attributes if it exists.

marcogalluzzi committed:
LPS-156216 Added Action Menu in article view

marcogalluzzi committed:
LPS-156216 Fixed redirect page on deletion

Kvale1733 committed:
LRQA-63520 Use 2 server cluster instead of 3

drewbrokke committed:
LPS-163706 account-service: schema: adds remaining workflow fields

drewbrokke committed:
LPS-163706 account: autogenerated: buildService

AFTER: adds remaining workflow fields

drewbrokke committed:
LPS-163706 account-service: adds upgrade process

Commit:
LPS-158949 Select id from change role

Commit:
LPS-158949 Set selected AccountRole item

Commit:
LPS-158949 Change function to getRaysourceContactRoleName

drewbrokke committed:
LPS-163706 account-service: adds more robust support for status updates and workflow instances

drewbrokke committed:
LPS-163706 account: autogenerated: buildService

AFTER: adds more robust support for status updates and workflow instances

drewbrokke committed:
LPS-163706 account-service: adds AccountEntryWorkflowHandler

Commit:
LPS-158949 Remove navigate if HTTP status was 404

Commit:
LPS-158949 Fix context for Liferay requests

Commit:
LPS-158949 Add mutations for update roles of userAccount

Commit:
LPS-158949 Handle edit userAccount row

Commit:
LPS-158949 Handle custom errors from server

drewbrokke committed:
LPS-163706 account-service: remote service: populate the ServiceContextThreadLocal with a userId for the activate and deactivate methods

drewbrokke committed:
LPS-163706 account-service: if the status has not changed, make it a no-op

4lejandrito committed:
LPS-166449 Injecting a headlessHttpClient into templates
LPS-166449 
LPS-166449 Sample usage:
LPS-166449 
LPS-166449 <#list headlessHttpClient.get("/headless-delivery/v1.0/sites/20121/blog-postings?sort=dateCreated:asc&filter=creatorId+eq+20125").items as blogPosting>
LPS-166449 	<h1>${blogPosting.headline}</h1>
LPS-166449 </#list>
LPS-166449 
LPS-166449 Warning: When the home page renders a template using this, if the server
LPS-166449 is restarted it will never start again. I don't really know why.

Commit:
LPS-158949 Simplify code

Commit:
LPS-158949 Order variable

Commit:
LPS-158949 Improve use of importDate

Commit:
LPS-158949 Fix conditional to create importDate value

weller0246 committed:
LPS-164261 Change Click commands to working assertNotChecked functions

weller0246 committed:
LPS-164261 Change Click commands to working functions in LAR.macro

weller0246 committed:
LPS-164261 Change Click commands to working functions in Staging.macro

marcogalluzzi committed:
LPS-165029 Computing selected node ancestors

Commit:
LPS-158949 Refactor useSearchTerm to use also in TeamMembers

Commit:
LPS-158949 Fix InviteTeamMembersForm looping

Commit:
LPS-158949 Create useSupportSeatsCount to avoid to change the count after searching

Commit:
LPS-158949 Fix getProjectRoles to use fetchPolicy as network-only

Commit:
LPS-158949 Add Header to Team Member Table

Commit:
LPS-158949 Move user to home if doesn't exist a externalReferenceCode

Commit:
LPS-158949 Avoid calculate support seats if it's searching

noornajj committed:
LPS-161035 Add service to find publication history by their modelClassNameId and modelClassPK

marcogalluzzi committed:
LPS-156216 Improved article selection criteria for redirect URL

marcogalluzzi committed:
LPS-165029 Renaming

marcogalluzzi committed:
LPS-156216 Fixed redirect URL when ancestor of selected node is deleted in navigator

marcogalluzzi committed:
LPS-165029 Fixed redirect URL for folder deletion

marcogalluzzi committed:
LPS-160407 Fixed selectedItemId for an article

marcogalluzzi committed:
LPS-156216 Added History action to article

drewbrokke committed:
LPS-163706 account-admin-web: removes active switch from the UI

The reasoning for this is that there is not really a use-case for creating an
inactive account, and activation and de-activation can already be done via an
action in the accounts list.

drewbrokke committed:
LPS-163706 account-service: log whenever the status parameter is ignored

Commit:
LPS-158949 Avoid custom styling

Commit:
LPS-158949 Fix div elements

Commit:
LPS-158949 Merge updating state with removing

Commit:
LPS-158949 Fix height for setup CP

drewbrokke committed:
LPS-163706 account-service: adds javadocs for the add and update methods

The key addition with this change is to document that when workflow is enabled,
the status parameter is ignored.

Commit:
LPS-158949 Fix approach to add new team member and filtering

drewbrokke committed:
LPS-163706 account: autogenerated: buildService

AFTER: adds javadocs

marcogalluzzi committed:
LPS-165029 FS

marcogalluzzi committed:
LPS-156216 Added icon to History action

marcogalluzzi committed:
LPS-156216 Added Print action to article

marcogalluzzi committed:
LPS-156216 Removed all article actions from portlet menu

drewbrokke committed:
LPS-163706 account-service: adds helper methods to AccountEntryOrganizationRelImpl

drewbrokke committed:
LPS-163706 account: autogenerated: buildService

AFTER: adds helper methods to AccountEntryOrganizationRelImpl

drewbrokke committed:
LPS-163706 account-service: adds helper methods to AccountEntryImpl

drewbrokke committed:
LPS-163706 account: autogenerated: buildService

AFTER: adds helper methods to AccountEntryImpl

drewbrokke committed:
LPS-163706 account-admin-web: refactors the AccountEntryDisplay class to extend the AccountEntryWrapper

drewbrokke committed:
LPS-163706 account-admin-web: handles more filter cases in the search container factory

drewbrokke committed:
LPS-163706 account-admin-web: adds additional filter items when workflow is enabled

drewbrokke committed:
LPS-163706 account-admin-web: refactors building dropdown items for clarity

drewbrokke committed:
LPS-163706 account-admin-web: updates AccountEntryDisplay creation usages

drewbrokke committed:
LPS-163706 account-admin-web: now that the display extends the model itself, we can use it directly

drewbrokke committed:
LPS-163706 account-admin-web: uses boolean status methods instead of comparisons

drewbrokke committed:
LPS-163706 account-admin-web: updates logo getter usages

drewbrokke committed:
LPS-163706 account-admin-web: updates domain getter usages

drewbrokke committed:
LPS-163706 account-admin-web: updates person account entry user getter usages

drewbrokke committed:
LPS-163706 account-admin-web: updates jsp imports

drewbrokke committed:
LPS-163706 account-service: adds AccountEntry asset renderer and factory

wanderlast committed:
LPS-165964 - set opacity to 1 for editable portlets to address accessibility concerns related to contrast

marcogalluzzi committed:
LPS-156216 Moved variable declaration to block where is used

marcogalluzzi committed:
LPS-165029 Removed unnecessary curly braces and indentation level

marcogalluzzi committed:
LPS-165029 Removed unnecessary short methods

drewbrokke committed:
LPS-166522 LPS-163706 account-admin-web: adds an all filter to the account table

drewbrokke committed:
LPS-163706 account-service: removes any workflow instance links on AccountEntry deletion

drewbrokke committed:
LPS-163706 account-service: uses boolean status method instead of comparison

shuyangzhou committed:
LPS-166679 No need to set ProxyModeThreadLocal to false for upgrade, this was added by 957eac055e5f14cf65c20754f79ec4f4ecc7ada4 to ensure async indexing during upgrade. But it is no longer needed since 8fa1895786654c2006a2b5537331582da850cf75 disabled indexing during upgrade.

shuyangzhou committed:
LPS-166679 Since 4107ff86 ProxyModelThreadLocal is true by default, no need to set to true explicitly.

shuyangzhou committed:
LPS-166679 SF

shuyangzhou committed:
LPS-166679 Not used anymore

shuyangzhou committed:
LPS-166679 Search does not use messaging proxy anymore, this threadlocal swapping is not used by anyone. This was added by cd1e1933 as a workaround for a customer's borken ES server setup. They should just fix their broken ES server rather than having us providing this flag, which would lead to unpredictable indexing results status.

shuyangzhou committed:
LPS-166679 ProxyModeThreadLocal.isForceSync() is always true.

shuyangzhou committed:
LPS-166679 Simplify and remove unused parameter

shuyangzhou committed:
LPS-166679 Not used anymore

shuyangzhou committed:
LPS-166679 Remove MessagingProxy usage for AuditRouter

shuyangzhou committed:
LPS-166679 Remove MessagingProxy usage for RulesEngine

shuyangzhou committed:
LPS-166679 Remove MessagingProxy usage for SchedulerEngine

Commit:
LPS-158949 Centralize remote app properties

Commit:
LPS-158949 SF

Commit:
LPS-165775 Redirection error when saving page SEO settings

david-truong committed:
LPS-161035 buildService

david-truong committed:
LPS-161035 create new taglib for publication history

david-truong committed:
LPS-161038 apply to journal-web

david-truong committed:
LPS-161035 buildLang

izaera committed:
LPS-165925 Migrate frontend-js-metal-web to ESM

izaera committed:
LPS-165925 Use newer npm-scripts

marcogalluzzi committed:
LPS-165029 Refactor - flattening if statements

Commit:
LPS-161364 Use new pattern for batch engine

danvb112 committed:
LPS-166235 Fix visual bug in DDM MultipleSelect trigger

4lejandrito committed:
LPS-166449 Using request dispatcher
LPS-166449 
LPS-166449 By doing this we get 3 nice things:
LPS-166449 
LPS-166449 1. The issue with the previous commit disappears ant the portal starts
LPS-166449    up and renders the home page.
LPS-166449 2. We gain some performance because we don't make a new request.
LPS-166449 3. We don't have to worry about cookies or authentication because we are
LPS-166449    reusing the incoming request.
LPS-166449 
LPS-166449 * I had to add the request wrapper to force json as return type.
LPS-166449 * I had to add the response wrapper to avoid breaking the whole page. I
LPS-166449 overrode all the methods that modify the response because we just care
LPS-166449 about the body. Without this the json is generated and injected into the
LPS-166449 template but the page won't render at all.

marcogalluzzi committed:
LPS-165029 Refactor - using single if statement

izaera committed:
LPS-165925 Fix dynamic exports

See https://github.com/liferay-frontend/liferay-portal/pull/2802#issuecomment-1285616488
and following messages.

A copy of the messages follow for faster reference:

----

Interestingly this fails in the CI because:

```
     [exec] Unable to require('metal-component'): please consider specifying the exported symbols explicitly in your 'npmscripts.config.js' file.
     [exec]
     [exec] ReferenceError: document is not defined
     [exec]     at Function.checkAnimationEventName_ (/opt/dev/projects/github/liferay-portal/modules/node_modules/metal-dom/lib/features.js:55:34)
```

That means that `metal-component` cannot be `require`d from `node`, but only from
the browser.

This makes finding the exports signature impossible for `npm-scripts` and forces
us to explicitly list them in the `npmscripts.config.js` which will be error prone.
Since `metal-component` is almost frozen, we may go this way, but I would like to
know if there's some way to make `metal-component` requireable from `node`.

----

Apparently there's
[a check in metal-dom](https://github.com/metal/metal.js/blob/master/packages/metal-dom/src/events.js#L54)
that makes it behave differently when it is in browser than when it is in the
server. I will try to find a way to make it think it's in the browser to make it
export the correct objects.

----

Seeing the
[server check code](https://github.com/metal/metal.js/blob/6002ec63153bca307859a5dfdfda0affb0274d49/packages/metal/src/coreNamed.js#L294-L310)
it seems to me that the most civilized way to make this work is to set some
environment variable when `npm-scripts` requires a package inside the build. Any
other solution I can think of seems a hack.

What we'll do is adding a way to define env vars to `npmscripts.config.js` for
scenarios where `symbols: 'auto'` is used with one package like this one.

----

So, the solution I explained above is very good but it will lead to a failure of
the build because precisely, what we want is `metal-dom` to think it is being
required from a browser and that is what makes the build fail.

So we need:

1. The env vars in `npmscripts.config.js` to fool `metal-dom` in any type of
   build and make it think it's in the browser.
2. Either:
    a. Changes in metal to prevent it from failing if `document` does not exist
    b. Changes in npm-scripts to polyfill the browser

Given that this is getting too complex and I don't think this can be generalized
for the next package we encounter with this issue, I think I'll simply list the
exports explicitly in the `npmscripts.config.js` file. Metal is stable enough so
as not to change the exports any more and, even if it does, finding and fixing
the error would be cheap.

----

adolfopa committed:
LPS-165496 Remove suggestions from template preview

adolfopa committed:
LPS-165650 Add "folderId" attribute to browser tag

adolfopa committed:
LPS-165650 Restrict breadcrumbs/search to root folder

Commit:
LPS-165576 Create BusinessType for MultiSelectPicklist Field

Commit:
LPS-165576 Add feature flag

drewbrokke committed:
LPS-163706 account-test: adds AccountEntryImplTest

drewbrokke committed:
LPS-163706 account-test: adds AccountEntryOrganizationRelImplTest

shuyangzhou committed:
LPS-166679 Semantic versioning

shuyangzhou committed:
LPS-166679 Remove MessagingProxy usage for Workflow

shuyangzhou committed:
LPS-166679 Remove leftover MessagingProxy from QuerySuggester

shuyangzhou committed:
LPS-166679 Remove messaging proxy support

drewbrokke committed:
LPS-163706 account-test: updates test coverage in AccountEntryLocalServiceTest

seltong committed:
LPS-166376 Create pow ddm function

seltong committed:
LPS-166376 Create pow function unit tests

Commit:
LPS-166241 Create Page Organization Detail

joelrocha68 committed:
LPS-160585 Fix tests and comments

joelrocha68 committed:
LPS-160585 Add AdminCanNotAddDuplicateRoutine test

joelrocha68 committed:
LPS-160585 SF

jkappler committed:
LPS-161165 Remove FF

jkappler committed:
LPS-161165 Add actions behind its own FF

jkappler committed:
LPS-161165 Remove unused view

marcogalluzzi committed:
LPS-165029 Refactor - using existing method

victorg1991 committed:
LPS-165426 Close the modal always when pressing the done button

4lejandrito committed:
LPS-166449 Caching servlet context
LPS-166449 
LPS-166449 https://github.com/liferay-headless/liferay-portal/pull/819#discussion_r1001914257

victorg1991 committed:
LPS-166451 Remove FF

victorg1991 committed:
LPS-166451 Auto SF

victorg1991 committed:
LPS-166174 Add also the masterLayout modifiedDate if present, otherwise changes in the master layout are not loaded

Commit:
POSHI-422 Add assertElementAccessible api

Commit:
POSHI-422 Add assertElementAccessible impl

claraizquierdo committed:
LPS-161165 Change frontend tests

adolfopa committed:
LPS-166432 Show article in display page if available

victorg1991 committed:
LPS-166296 Adjust focus when opening/closing sidebar panels

When opening a tab move the focus to the sidebarpanel, and when closing it move the focus back to the button

Commit:
POSHI-423 Update axe-selenium version

achaparro committed:
LPS-166223 No need to wrap too soon the dataSource since we are creating new tables in the default partition, otherwise some DBPartition checks can fail because Spring (class InsfrastructureUtil) is not ready

Commit:
POSHI-423 Update wcag version

rebsilva committed:
LPS-164922 Sort the classes inside detectClickOutside function

victorg1991 committed:
LPS-166451 Prevent null

In the global site the friendlyUrl input is not available

adolfopa committed:
LPS-166518 Add empty state message

tinycarol committed:
LPS-165825 - Move focus to ResultsBar if it renders

joshitagaki committed:
LRAC-11952 Fix url to go to admin page

thaiscabrals committed:
LPS-166528 Remove tests that failed because of the quarantine bug

pat270 committed:
LPS-165964 portlet-content-editable should have better contrast (accessibility)

pat270 committed:
LPS-165964 liferay-learn-site-initializer portlet-content-editable should have better contrast (accessibility)

kungfumanda committed:
LPS-165654 portal-language-lang: Add help text for the "Select Vocabularies" feature when enabled

https://issues.liferay.com/browse/LPS-165654

kungfumanda committed:
LPS-165654 portal-search-web: Change "Select Vocabularies" configuration help text when it's enabled

https://issues.liferay.com/browse/LPS-165654

david-truong committed:
LPS-161035 create service provider to get publication history for versioned assets that don't follow typical patterns like journal

BryanEngler committed:
LPS-164718 LPS-166563 portal-search-admin-web: Add button in Index Actions to reindex text embeddings

https://issues.liferay.com/browse/LPS-166563

kungfumanda committed:
LPS-165654 portal-search-web: Update learn-message config and alphabetize the class names

https://issues.liferay.com/browse/LPS-165654

RenanUmV committed:
LPS-166543 Remove immediate from diff module

RenanUmV committed:
LPS-166543 Remove immediate from Invitation module

RenanUmV committed:
LPS-166543 Remove immediate from Learn module

RenanUmV committed:
LPS-166543 Remove immediate from Microsoft Translator module

RenanUmV committed:
LPS-166543 Remove immediate from Normalizer module

RenanUmV committed:
LPS-166543 Remove immediate from Plugins Admin module

RenanUmV committed:
LPS-166543 Remove immediate from Words module

RenanUmV committed:
LPS-166543 Remove immediate from Xstream module

ealonso committed:
LPS-166548 Language key

ealonso committed:
LPS-166548 buildLang

kenjiheigel committed:
POSHI-423 Use com.liferay.poshi.core:1.0.78

danvb112 committed:
LPS-166526 Make MultiSelect Picklist available in filter modal in Views

BryanEngler committed:
LPS-164718 LPS-166563 portal-language-lang: Add language key

https://issues.liferay.com/browse/LPS-166563

BryanEngler committed:
LPS-164718 LPS-166563 portal-language-lang: buildLang

https://issues.liferay.com/browse/LPS-166563

brenobcos committed:
LPS-165936 Homepage My watched tags

brenobcos committed:
LPS-166484 Modified sort order

brenobcos committed:
LPS-166487 Updated request All Questions

brenobcos committed:
LPS-166485 updated useQuestionsSections

brenobcos committed:
LPS-166488 Refactoring QuestionList

Commit:
LPS-150588 Add Macro on CP2Util

brenobcos committed:
LPS-166485 SF

Commit:
LPS-150588 Add Path on CP2Utils

Commit:
LPS-150588 add ClickProjectGoesToProjectPage test

Commit:
LPS-150588 add SearchUpdatedWithSearchTerm test

Commit:
LPS-140176 Fix test

patriciajeanperez committed:
LPS-163706 Functional Test - Rewrite test not to interact with active switch

patriciajeanperez committed:
LPS-163706 Functional Test - No longer needed

patriciajeanperez committed:
LPS-163706 Functional Test - Remove macros that interacts with active switch

RenanUmV committed:
LPS-166543 Semver

patriciajeanperez committed:
LPS-163706 Functional Test - Add coverage for activating an inactive account

SelenaAungst committed:
LPS-166594 Added aria-label to options button.

Commit:
LPS-166389 fix ISO 3166 country codes for default countries

Signed-off-by: Vy-CES <vy.bui@gs.liferay.com>

Commit:
LPS-166447 Add case to view role of page template cards

lfbesada committed:
LPS-166578 Use the proper className for the info framework implementation

lfbesada committed:
LPS-166578 SF inline

JessieZhao0714 committed:
LPS-166278 Add testcase to verify JWKS

lfbesada committed:
LPS-166616 SF

lfbesada committed:
LPS-166616 Use InfoSearchClassMapper

lfbesada committed:
LPS-166616 SF rename

lfbesada committed:
LPS-166616 SF reuse

gaborlovasdotcom committed:
LPS-166278 SF

veroglez committed:
LPS-166564 Add aria-label for the panel and role region

veroglez committed:
LPS-166564 Add aria-live for the ClayAlert to notify in a polite way the message

veroglez committed:
LPS-166564 BuildLang

boton committed:
LPS-166625 Add a z-index to move to the foreground above content

weller0246 committed:
LPS-164621 Change locator to correct one

danvb112 committed:
LPS-166235 Fix frontend snapshot test

Commit:
LPS-158949 SF

Commit:
LPS-166667 Blocking guest user access to EVP forms

thaiscabrals committed:
LPS-155711 Remove tests that failed because of the quarantine bug

brianchandotcom committed:
LPS-166449 SF

brianchandotcom committed:
LPS-166449 Add TODO to limit path

sandrodw3 committed:
LPS-166655 Add handlerRef only in desktop

kungfumanda committed:
LPS-165654 portal-language-lang: gw buildLang

https://issues.liferay.com/browse/LPS-165654

Commit:
LPS-166208 Create Policy Auto Details page

pyoo47 committed:
LRCI-3316 Add timeout and retry to git clone attempt

brookedalton committed:
LPS-166390 Add db partitioning publications suite

brookedalton committed:
LPS-166698 Remove upgrade tests from search db partition suite

david-truong committed:
LPS-161038 add feature flag

xbrianlee committed:
LPS-166698 Remove unnecessary parenthesis

kenjiheigel committed:
POSHI-423 Update axe accessibility property

kenjiheigel committed:
POSHI-423 Use new axe dependencies and implementation

michaelhashimoto committed:
LRCI-3231 Add ability to find the portal branch name from a Testray Build

michaelhashimoto committed:
LRCI-3231 Record the test count, testray url, & the portal branch in the CI History json file

michaelhashimoto committed:
LRCI-3231 Remove unused upstream failure util methods

kenjiheigel committed:
POSHI-422 Consolidate logic

michaelhashimoto committed:
LRCI-3231 Add JobHistory object to be related to the Job objects

michaelhashimoto committed:
LRCI-3231 Add testray case result URL & the test count to the TestHistory objects

michaelhashimoto committed:
LRCI-3231 Relate the Job objects to the JobHistory objects

michaelhashimoto committed:
LRCI-3231 Add caching for the batch test class group objects within Job objects

michaelhashimoto committed:
LRCI-3231 Remove unused cucumber classes

brianchandotcom committed:
LPS-166449 Rename/SF

lfbesada committed:
LPS-166735 Use a PortletActionRequest most complete mock

lfbesada committed:
LPS-166735 SF unused

adolfopa committed:
LPS-165650 Baseline

brianchandotcom committed:
LPS-163706 I don't see the 'Required.' 'Optional.' pattern in our existing javadocs

brianchandotcom committed:
LPS-163706 Regen

brianchandotcom committed:
LPS-163706 Wordsmith

brianchandotcom committed:
LPS-163706 Wordsmith

brianchandotcom committed:
LRCI-3231 prep next

brianchandotcom committed:
LPS-166278 Tabs vs. spaces

Commit:
LPS-166437 Add macros to assert element focused or not

Commit:
LPS-166437 Add case to focus on Skip to Product Menu when product menu open

brianchandotcom committed:
POSHI-422 prep next

brianchandotcom committed:
LPS-161035 Not needed

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-161035 Wordsmith

brianchandotcom committed:
LPS-161035 Regen

brianchandotcom committed:
LPS-156216 LPS-165029 LPS-160407 Rename

brianchandotcom committed:
LPS-156216 LPS-165029 LPS-160407 SF

brianchandotcom committed:
LPS-166735 SF

Italo-gabriel committed:
LPS-165048 fix path entry kebab menu - view entry eye button and remove quarantine

brianchandotcom committed:
LPS-165651 Wordsmith

brianchandotcom committed:
LPS-165651 Regen

brianchandotcom committed:
LPS-166376 SF

Commit:
LPS-165576 buildRest

Commit:
COMMERCE-9603 - Remove delete permission from channel payment methods

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
